plugin/evm/vm_test.go

```diff
@@ -3181,26 +3181,3 @@ func TestCrossChainMessagestoVM(t *testing.T) {
 	require.NoError(err)
 	require.True(calledSendCrossChainAppResponseFn, "sendCrossChainAppResponseFn was not called")
 }
-
-func TestSignatureRequestsToVM(t *testing.T) {
```

Comment:
Decided to remove this test as the primary goal of this test previously was in respect to a `NoopSignatureRequestHandler`. Now that the handler is initialized this test won't be helping, and we already check for expected responses in the signature request handler tests.

Reply:
Can we update this test to check that the VM calls `AppResponse` correctly with a known and unknown `messageID` instead of removing it?

Reply:
updated the test for the above.

Reply:
Can you update this test so that it asserts the expected response is sent by the VM instead of just checking that it can be unmarshalled correctly?

Reply:
updated

Reply:
Nice, this is perfect